api/api.go

```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
 	w.Write(b)
 
 }
+
+func getMembers(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.Query()
+	limit := viper.GetInt("api.event_query_limit")
+	servers := viper.Get("discord.servers").(*config.Servers)
+	queryAmount, exists := query["q"]
+	if !exists || len(query) == 0 {
+		http.Error(w, "Please add the parameter 'q'", 403)
```

Comment:
400 (bad request) is probably more suited than 403 code

Reply:
I was just following the codes as stated in previous API requests, I can always just send another PR request to fix all the codes

Reply:
Depends what @ericm says, not sure why they'd all be 403